Merchant/Core/Extensions/Double+Extensions.swift

```diff
@@ -20,4 +20,29 @@ extension Double {
     func toInt() -> Int {
         return Int(self)
     }
+    
+    func toCurrency() -> String {
+        let f = NumberFormatter()
+        f.locale = Locale.current
+        let selectedCurrency = UserManager.shared.selectedCurrency
+        let locales = RateManager.shared.currencyToCountryLocales?[selectedCurrency.ticker]
+        if let locales = locales {
+            if locales.count > 0 {
+                let locales = locales[0].locales.components(separatedBy:",")
+                if (locales.count > 0) {
+                    let locale = locales[0].replacingOccurrences(of: "-", with: "_")
```

Comment:
Im confused which 'locales' variable is used here now. we can also avoid nested if by doing a guard statement as mentioned by JB above